test/multiverse/suites/active_record_pg/active_record_test.rb

```diff
@@ -414,6 +414,24 @@ def test_metrics_for_direct_sql_select
     assert_generic_rollup_metrics('select')
   end
 
+  def test_metrics_for_find_by_sql
+    skip unless active_record_major_version >= 7
```

Comment:
If there's time / interest for a side project, we could open a PR against `rubocop-minitest` to treat `skip` the same way that `rubocop` treats `raise` (and require a blank line after it appears). No need to make that change here before that PR though.

Reply:
Went ahead and added the blank line. I love the contribution idea too!